### ISSUE TITLE
6.10.0  build  failed

### ISSUE THREAD

**Issue by zuowenjian** [Geal/nom#1285]
- Rust version : 1.50.0
- nom version : 6.1.0
- nom compilation features used: default

[fail link](https://github.com/zuowenjian/nom/runs/1897953824?check_suite_focus=true)

```
    Running `rustc --crate-name lexical_core --edition=2018 /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/lexical-core-0.7.5/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=2 --cfg 'feature="arrayvec"' --cfg 'feature="correct"' --cfg 'feature="default"' --cfg 'feature="ryu"' --cfg 'feature="static_assertions"' --cfg 'feature="std"' --cfg 'feature="table"' -C metadata=431beadc60849397 -C extra-filename=-431beadc60849397 --out-dir /home/runner/work/nom/nom/target/debug/deps -L dependency=/home/runner/work/nom/nom/target/debug/deps --extern arrayvec=/home/runner/work/nom/nom/target/debug/deps/libarrayvec-ea8c77fe5b6fcc41.rmeta --extern bitflags=/home/runner/work/nom/nom/target/debug/deps/libbitflags-de31228f2c0549ed.rmeta --extern cfg_if=/home/runner/work/nom/nom/target/debug/deps/libcfg_if-d836a7202f0431f0.rmeta --extern ryu=/home/runner/work/nom/nom/target/debug/deps/libryu-37b6e9a22bb2209d.rmeta --extern static_assertions=/home/runner/work/nom/nom/target/debug/deps/libstatic_assertions-ac453ec36b9d397e.rmeta --cap-lints allow --cfg limb_width_64`
error[E0034]: multiple applicable items in scope
   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/bitvec-0.19.4/src/field.rs:307:25
    |
307 |         check("load", len, M::BITS);
    |                               ^^^^ multiple `BITS` found
```

**Comment by qiuzhanghua** [comment]
It seems that bitvec should upgrade to version >=0.21. @Geal

**Comment by Geal** [comment]
thank you for the report, I just released nom 6.1.1 with the fix in #1286
